lib/v2/npr/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/package/:name': ['bennyyip'],
```

Comment:
```suggestion
    '/:endpoint?': ['bennyyip'],
```
https://github.com/DIYgod/RSSHub/blob/b569a91df4b592bfe69e0b2a4a5dd6eb0331b740/lib/v2/npr/router.js#L2